ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/tenant_evpn_vxlan/leaf-tenant-vlan-interfaces.j2

```diff
@@ -21,19 +38,36 @@
 {%                          endif %}
 {%                     endfor %}
 {%                 endif %}
-{%                 if tenants[tenant].vrfs[vrf].svis[svi].ip_virtual_router_address is defined %}
-    ip_virtual_router_address: {{ tenants[tenant].vrfs[vrf].svis[svi].ip_virtual_router_address | ipaddr('address') }}
+{# Virtual Router IP Address #}
+{%                 if profile.per_svi.ip_virtual_router_address is defined or profile.svi_profile.ip_virtual_router_address is defined %}
+    ip_virtual_router_address: {{ profile.per_svi.ip_virtual_router_address if profile.per_svi.ip_virtual_router_address is defined else profile.svi_profile.ip_virtual_router_address }}
```

Comment:
ip_virtual_router_address needs to render only the address and not the mask
update logice to include ipaddr filter: {{ profile.per_svi.ip_virtual_router_address | ipaddr('address') if profile.per_svi.ip_virtual_router_address is defined else profile.svi_profile.ip_virtual_router_address | ipaddr('address') }}